ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -44,7 +44,7 @@ StatusSectionLayout {
     property Component membershipRequestPopup
 
     readonly property string filteredSelectedTags: {
-        if (!community || !community.tags)
+        if (!community || !community.tags || !!community.tags)
```

Comment:
`!community.tags || !!community.tags` looks like mutually exclusive conditions

Reply:
The `!community.tags` works when you have tags defined and it is empty 
The '!!community.tags` works when tags is undefined,
I specifically added this fix because I was getting a lot of errors  like this here
```
CommunitySettingsView.qml:51: TypeError: Cannot read property 'map' of null
CommunitySettingsView.qml:61: SyntaxError: JSON.parse: Parse error 
```
and the only thing which worked for me was using the form you see now in PR

Reply:
So the thing that looks off here is that the condition has two falsy conditions (`!community || !community.tags`) and one truthy condition (`!!cmmunity.tags`).

So you're saying, this condition is valid when either, `community` is falsy, or `community.tags` is falsy, or `community.tags` is truthy.

Reply:
Oh - yes - you both are correct.

Reply:
Interesting case - it actually needs `community.tags != "null"` there to ignore empty tags list